server/views/manage/outOfServiceBeds/premisesIndex.njk

```diff
@@ -18,58 +18,54 @@
 {% endblock %}
 
 {% block content %}
-    <div class="govuk-grid-row">
-        <div class="govuk-grid-column-full-width">
-            {% include "../../_messages.njk" %}
+    {% include "../../_messages.njk" %}
 
-            <span class="govuk-caption-l">
+    <span class="govuk-caption-l">
         <a href="{{ paths.premises.show({premisesId: premises.id}) }}">{{ premises.name }}</a>
-      </span>
+    </span>
 
-            <h1 class="govuk-heading-l">{{ pageHeading }}</h1>
+    <h1 class="govuk-heading-l">{{ pageHeading }}</h1>
 
-            {{ govukSummaryList({
-                classes: 'govuk-summary-list--no-border',
-                rows: [            {
-                    key: {
-                        text: "Total out of service beds matching filters"
-                    },
-                    value: {
-                        text: OutOfServiceBedUtils.outOfServiceBedCount(totalResults)
-                    }
-                }
-                ]
-            }) }}
+    {{ govukSummaryList({
+        classes: 'govuk-summary-list--no-border',
+        rows: [            {
+            key: {
+                text: "Total out of service beds matching filters"
+            },
+            value: {
+                text: OutOfServiceBedUtils.outOfServiceBedCount(totalResults)
+            }
+        }
+        ]
+    }) }}
 
-            {{ mojSubNavigation({
-                label: 'Sub navigation',
-                items: [{
-                    text: 'Current',
-                    href: paths.outOfServiceBeds.premisesIndex({premisesId: premises.id, temporality: 'current'}),
-                    active: temporality === 'current'
-                }, {
-                    text: 'Future',
-                    href: paths.outOfServiceBeds.premisesIndex({premisesId: premises.id, temporality: 'future'}),
-                    active: temporality === 'future'
-                }, {
-                    text: 'Historic',
-                    href: paths.outOfServiceBeds.premisesIndex({premisesId: premises.id, temporality: 'past'}),
-                    active: temporality === 'past'
-                }]
-            }) }}
+    {{ mojSubNavigation({
+        label: 'Sub navigation',
+        items: [{
+            text: 'Current',
+            href: paths.outOfServiceBeds.premisesIndex({premisesId: premises.id, temporality: 'current'}),
+            active: temporality === 'current'
+        }, {
+            text: 'Future',
+            href: paths.outOfServiceBeds.premisesIndex({premisesId: premises.id, temporality: 'future'}),
+            active: temporality === 'future'
+        }, {
+            text: 'Historic',
+            href: paths.outOfServiceBeds.premisesIndex({premisesId: premises.id, temporality: 'past'}),
+            active: temporality === 'past'
+        }]
+    }) }}
```

Comment:
Could move the generation of all this into the controller to avoid repetition of base path, like in the `premisesController`